ts/screens/authentication/__tests__/NewOptInScreen.test.tsx

```diff
@@ -5,9 +5,9 @@ import { applicationChangeState } from "../../../store/actions/application";
 import { appReducer } from "../../../store/reducers";
 import I18n from "../../../i18n";
 import { renderScreenWithNavigationStoreContext } from "../../../utils/testWrapper";
-import NewOptInScreen from "../NewOptInScreen";
+import OptInScreen from "../OptInScreen";
```

Comment:
What about renaming the test file too in `OptInScreen.test.tsx`?  What do you think?

Reply:
Yes! Good catch! thanks!